changelogs/fragments/750-jc-new-filter.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- jc - new filter to convert the output of many shell commands and file-types to JSON. Uses the jc library at https://github.com/kellyjonbrazil/jc. For example, filtering the STDOUT output of ``uname -a`` via ``{{ result.stdout | community.general.jc('uname') }}``
```

Comment:
```suggestion
- jc - new filter to convert the output of many shell commands and file-types to JSON. Uses the jc library at https://github.com/kellyjonbrazil/jc. For example, filtering the STDOUT output of ``uname -a`` via ``{{ result.stdout | community.general.jc('uname') }}``. Requires Python 3.5+ (https://github.com/ansible-collections/community.general/pull/750).
```

Reply:
(3.5 is the minimal Python 3 version supported by Ansible)

Reply:
There are a bunch of f-strings littered about `jc` so I believe only python 3.6+ is supported. I'll update to 3.6.